dts/arm/infineon/psoc6/psoc6_02/psoc6_02.124-bga.dtsi

```diff
@@ -0,0 +1,2051 @@
+				label = "P5";
+				#gpio-cells = <2>;
+			};
+			gpio_prt6: gpio@40310300 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310300 0x80>;
+				interrupts = <6 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P6";
+				#gpio-cells = <2>;
+			};
+			gpio_prt7: gpio@40310380 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310380 0x80>;
+				interrupts = <7 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P7";
+				#gpio-cells = <2>;
+			};
+			gpio_prt8: gpio@40310400 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310400 0x80>;
+				interrupts = <8 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P8";
+				#gpio-cells = <2>;
+			};
+			gpio_prt9: gpio@40310480 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310480 0x80>;
+				interrupts = <9 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P9";
+				#gpio-cells = <2>;
+			};
+			gpio_prt10: gpio@40310500 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310500 0x80>;
+				interrupts = <10 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P10";
+				#gpio-cells = <2>;
+			};
+			gpio_prt11: gpio@40310580 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310580 0x80>;
+				interrupts = <11 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P11";
+				#gpio-cells = <2>;
+			};
+			gpio_prt12: gpio@40310600 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310600 0x80>;
+				interrupts = <12 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P12";
+				#gpio-cells = <2>;
+			};
+			gpio_prt13: gpio@40310680 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310680 0x80>;
+				interrupts = <13 6>;
+				gpio-controller;
+				ngpios = <8>;
+				status = "disabled";
+				label = "P13";
+				#gpio-cells = <2>;
+			};
+			gpio_prt14: gpio@40310700 {
+				compatible = "infineon,mtbhal-gpio";
+				reg = <0x40310700 0x80>;
+				interrupts = <14 6>;
+				gpio-controller;
+				ngpios = <2>;
+				status = "disabled";
+				label = "P14";
+				#gpio-cells = <2>;
+			};
+
+			/* audioss_clk_i2s_if */
+			p5_0_audioss0_clk_i2s_if: p5_0_audioss0_clk_i2s_if {
+				DT_CAT1_PINCTRL_INFO(5, 0, HSIOM_SEL_ACT_10, AUDIOSS_CLK_I2S_IF);
+			};
+			p9_0_audioss0_clk_i2s_if: p9_0_audioss0_clk_i2s_if {
+				DT_CAT1_PINCTRL_INFO(9, 0, HSIOM_SEL_ACT_10, AUDIOSS_CLK_I2S_IF);
+			};
+			p11_0_audioss1_clk_i2s_if: p11_0_audioss1_clk_i2s_if {
+				DT_CAT1_PINCTRL_INFO(11, 0, HSIOM_SEL_ACT_10, AUDIOSS_CLK_I2S_IF);
+			};
+			p13_0_audioss1_clk_i2s_if: p13_0_audioss1_clk_i2s_if {
+				DT_CAT1_PINCTRL_INFO(13, 0, HSIOM_SEL_ACT_10, AUDIOSS_CLK_I2S_IF);
+			};
+
+			/* audioss_pdm_clk */
+			p10_4_audioss0_pdm_clk: p10_4_audioss0_pdm_clk {
+				DT_CAT1_PINCTRL_INFO(10, 4, HSIOM_SEL_ACT_9, AUDIOSS_PDM_CLK);
```

Comment:
we also agreed on not using macros that hide actual DT code. Macros shall only be used for property content, as it's used e.g. in nrf or stm32/gd32 to encode `pinmux` property.

Reply:
done